### ISSUE TITLE
The error message for UniqueKeys contains the template variable, but not the actual value.

### ISSUE THREAD

**Issue by szilajka** [gregsdennis/json-everything#365]
**Environment**
- Nuget Package: JsonSchema.Net, JsonSchema.Net.UniqueKeys
- Nuget Version: (JsonSchema.Net) 3.3.2, (UnqiueKeys) 2.0.1
- OS: Windows 10, 11
- .Net Target: 6.0, 7.0

**Describe the bug**
When using the UniqueKeys keyword (from JsonSchema.Net.UniqueKeys nuget package) in the Json schema and validate a json (that contains duplicated values) with it, the error message contains the  **[[duplicates]]** template variable instead of the location of the duplicates.
The error message: _Found duplicates at the following index pairs: [[duplicates]]_.

**To Reproduce**
1. Create a JSON Schema that uses the unique keys vocabulary (I used the predefined schema that's in the UniqueKeys nuget package (I think it uses the 2020-12 schema)).
For example:
``` json
{
  "$schema": "https://gregsdennis.github.io/json-everything/meta/unique-keys",
  "type": "array",
  "items": {
    "type": "object",
    "properties": {
      "foo": {
        "type": "integer"
      }
    }
  },
  "uniqueKeys": [
    "/foo"
  ]
}
```
2. Create a JSON input that contains duplicated values.
For example:
``` json
[
  { "foo": 8 },
  { "foo": 12 },
  { "foo": 8 }
]
```
3. Validate the (wrong) JSON input against the JSON Schema.

**Expected behavior**
The error message should contain the position of duplicates.
For example: 
_Found duplicates at the following index pairs: (1, 2)_

**Additional context**
Used the following code to reproduce the issue:
```csharp
const string testJson = @"
[
  { ""foo"": 8 },
  { ""foo"": 12 },
  { ""foo"": 8 }
]";

Json.Schema.UniqueKeys.Vocabularies.Register();

var jsonSerializerOptions = new JsonSerializerOptions { WriteIndented = true };

var fooSchema = new JsonSchemaBuilder()
    .Type(SchemaValueType.Integer);
var innerObjectSchema = new JsonSchemaBuilder()
    .Type(SchemaValueType.Object)
    .Properties(("foo", fooSchema));
var finalSchema = new JsonSchemaBuilder()
    .Schema(Json.Schema.UniqueKeys.MetaSchemas.UniqueKeysId)
    .Type(SchemaValueType.Array)
    .Items(innerObjectSchema)
    .UniqueKeys("/foo")
    .Build();

var testJsonNode = JsonNode.Parse(testJson);
var validationResult = finalSchema.Validate(testJsonNode, new ValidationOptions { OutputFormat = OutputFormat.Basic, ValidateAs = Draft.Draft202012 });

var serializedValidationResult = JsonSerializer.Serialize(validationResult, jsonSerializerOptions);
Console.WriteLine(serializedValidationResult);
```

**Comment by gregsdennis** [comment]
Yeah, it looks like `uniqueKeys` is trying to use a `pairs` token instead.  I'll have a fix out for this soon.  Thanks for the report.

**Comment by gregsdennis** [comment]
So here's the thing:  I don't have a way to publish a bug fix for an older version, and this lib is already updated to work with the beta versions of JsonSchema.Net v4.  This is going to be published as v2.1.0-beta2.
